### ISSUE TITLE
Error when posting comments on question pages

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#2895]
Possibly related to Rails 5 upgrade and route changes... similar to #2884 and @Souravirus's fix at #2887 

Reported by @steviepubliclab !

**Comment by Souravirus** [comment]
Yeah same error which I posted on #2886 it will be corrected by fix on #2887

**Comment by jywarren** [comment]
Indeed! 

`https://publiclab.org/comment/create/16551?type=question&body=Testing!&_=1529693424592 404 (Not Found)`

**Comment by jywarren** [comment]
OK, i'll merge that and we'll see!

**Comment by Souravirus** [comment]
Yeah @jywarren

**Comment by jywarren** [comment]
working on that now

**Comment by Souravirus** [comment]
is there a need for tests for the changes done?

**Comment by jywarren** [comment]
Tnx for your help!!!!!

On Jun 22, 2018 3:29 PM, "Sourav Sahoo" <notifications@github.com> wrote:

is there a need for tests for the changes done?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/publiclab/plots2/issues/2895#issuecomment-399556856>,
or mute the thread
